github/misc.go

```diff
@@ -83,6 +83,61 @@ func (c *Client) ListEmojis(ctx context.Context) (map[string]string, *Response,
 	return emoji, resp, nil
 }
 
+// Conduct represents metric that contains name, key, url, body.
+type Conduct struct {
```

Comment:
The name and description here seem off.

I think we should give it a more explicit and long name. `Conduct` isn't clear in context of all the types present in the package.

What do you think of `CodeOfConduct` instead?

The comment should probably say that `// CodeOfConduct represents a code of conduct.` or so.